root/static/scripts/edit/components/ArtistCreditNameEditor.js

```diff
@@ -114,6 +115,20 @@ component _ArtistCreditNameEditor(
     });
   }
 
+  function handleMoveDown() {
+    dispatch({
```

Comment:
I was tempted to put these `dispatch` calls on single lines, but the rest of the file is using multiple lines so I did the same for consistency.